cookbook/src/tx_segwit-v0.md

```diff
@@ -255,8 +255,8 @@ Inside the [`TxIn`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction
    It should be empty. That's why the `ScriptBuf::new()`.
 - `sequence` is the sequence number; it is a [`Sequence`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Sequence.html) type.
    We are using the [`ENABLE_RBF_NO_LOCKTIME`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Sequence.html#associatedconstant.ENABLE_RBF_NO_LOCKTIME) constant.
-- `witness` is the witness stack; it is a [`Witness`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html) type.
-   We are using the [`default`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/transaction/struct.Witness.html#impl-Default) method to create an empty witness that will be filled in later after signing.
+- `witness` is the witness stack; it is a [`Witness`](https://docs.rs/bitcoin/latest/bitcoin/blockdata/witness/struct.Witness.html) type.
```

Comment:
These, and all the links, should be to v0.30.0 not to latest.

Reply:
Yes, I will update them. Good point

Reply:
Done in ea10b995bc81ce09758b542393b07ca982e299f0